resources/schemas/stsci.edu/asdf/time/time-1.2.0.yaml

```diff
@@ -178,24 +178,38 @@ definitions:
         do for the purposes of ASDF?]
 
     enum:
-      - iso
-      - yday
       - byear
-      - jyear
+      - cxcsec
       - decimalyear
+      - gps
+      - iso
       - jd
+      - jyear
       - mjd
-      - gps
       - unix
-      - cxcsec
+      - unix_tai
+      - yday
+
+  other_format:
+    description: |
+      The other formats supported by astropy.time:
+      https://docs.astropy.org/en/latest/time/index.html#time-format
+
+    enum:
+      - byear_str
+      - datetime
+      - fits
+      - isot
+      - jyear_str
+      - plot_date
+      - ymdhms
+      - datetime64
 
 anyOf:
   - $ref: "#/definitions/string_formats"
 
   - $ref: "#/definitions/array_of_strings"
 
-  - $ref: "../core/ndarray-1.0.0#/anyOf/1"
-
```

Comment:
Isn't this PR supposed to make it so that the removal of this becomes unnecessary?